roles/xnat/tasks/settings_files.yml

```diff
@@ -9,6 +9,15 @@
     force: true
   notify: Restart tomcat
 
+- name: "Validate xnat_config.site_name: {{ xnat_config.site_name }}"
+  ansible.builtin.assert:
+    that:
+      - xnat_config.site_name is match('^[A-Za-z][A-Za-z0-9_]*$')
```

Comment:
I checked the regex with https://regex101.com/r/LQsvzE/1. And the [assertion passed in CI](https://github.com/UCL-MIRSG/ansible-collection-infra/actions/runs/8408166400/job/23024000934?pr=84#step:5:1123)